### PR TITLE
[expo][docs] add sentry-expo to bundledNativeModules

### PR DESCRIPTION
**Pull request by cruzach**
# Why

This will make it so that future updates to sentry-expo don't always have to be compatible with all SDK versions (although hopefully they always are anyways 😅 )

### CODE REVIEW

**Comment by ide** [comment]
Looks good but waiting on review from someone more familiar with whether we want to include pure JS packages in here.

File: docs/pages/guides/using-sentry.md
Comment (brentvatne) [inline docs/pages/guides/using-sentry.md:43]:
```suggestion
  > If you're using SDK 39 or lower, run `npm install sentry-expo@~3.0.0`
```

Comment (brentvatne) [reply docs/pages/guides/using-sentry.md:43]: we can just say npm install, people using yarn will figure it out :)

**Review by brentvatne — APPROVED** [review]

**Pull request merged**
